openc3-cosmos-init/plugins/packages/openc3-tool-common/src/plugins/dialog/ConfirmDialog.vue

```diff
@@ -44,11 +57,19 @@ export default {
         title: 'Title',
         text: 'The text that is displayed',
         okText: 'Ok',
+        validateText: 'CONFIRM',
```

Comment:
This is a fine default.   Should you be asking for the SCOPE name when deleting a scope?

Reply:
Yes I do. That change is in Enterprise code.